### PR TITLE
Add support for V7 keepalive

### PR DESCRIPTION
**Pull request by karknu**
NodeToNode version 7 fixed the cbor encoding for the keepalive
miniprotocol.

### CODE REVIEW

**Comment by karknu** [comment]
Fixes #3187.

**Comment by jbgi** [comment]
tested successfully on alonzo-blue.

File: network-mux/demo/cardano-ping.hs [hunk 1]
```diff
@@ -401,15 +412,21 @@ pingClient tracer Options{quiet, magic, json, maxCount} peer = bracket
...
-    keepAlive bearer timeoutfn peerStr td cookie = do
+    keepAlive :: MuxBearer IO
+              -> TimeoutFn IO
+              -> String
+              -> NodeToNodeVersion
+              -> TDigest 5
+              -> Word32
+              -> IO ()
+    keepAlive _ _ _ _ _ cookie | cookie == maxCount = return ()
+    keepAlive bearer timeoutfn peerStr version td cookie = do
```
Comment (coot) [inline network-mux/demo/cardano-ping.hs:423]:
```suggestion
    keepAlive bearer timeoutfn peerStr version td !cookie = do
```

**Review by coot — CHANGES_REQUESTED** [review]

**Pull request merged**